### DISCUSSION TITLE
Where to get info about release cycle and dates?

### DISCUSSION THREAD

**Discussion by vitonsky** [eclipse-theia/theia#13935] [Q&A] [answered]
Hi, i've reported [a problem](https://github.com/eclipse-theia/theia/issues/13888) and as i see this problem been fixed quickly.

Now i wait a new release with fix, to test how it works, but i can't found any info about release cycle and plans about next patch release. Do you have one?

If yes, where i can check it. Or who i should email to getting know. Thanks.

**Comment by msujew** [comment, marked as answer]
Hey @vitonsky,

we generally release every month per our [releases page](https://theia-ide.org/releases/). Framework and IDE releases follow the same cadence. The release itself is usually on the last Thursday of the month, but that may vary sometimes due to vacations, etc.

It probably makes sense to have a patch release for the mentioned issue. @jfaltermeier WDYT?
